services/ecosystems/README.md

```diff
@@ -0,0 +1,63 @@
+# Biotablero Ecosystems Service
+
+Ecosystems service for biotablero
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+You'll need nodejs v14.15+ and yarn to run the project.
+
+### Install dependencies
+
+After cloning the project, install its dependencies running: `yarn install`
+
+### Setup
+
+Copy the [config file](config/default.json) with the name of the environment you're running, this name needs to be the same as the env var **NODE_CONFIG_ENV**.
+
+By default **NODE_CONFIG_ENV** is _develop_, so you'll need to create _config/develop.json_ and set minimum the "db" parameters.
+
+### Run
+
+Run `yarn start` to start the server, this will launch nodemon ready to watch your changes.
+
+## Deployment
+
+Deployment is made with docker, you'll need Docker v17.05.0+ and docker compose v1.17.1+
+
+### Build image
+
+To build the image run: `docker build -t biotablero-ecosystems:<version> .`
```

Comment:
There is no Dockerfile in this service, please add one and test it

Reply:
I have created, however it only works without this line(10) `RUN find /home/node/app/config/ ! -name 'default.json' -type f -exec rm -f {} +`
![Screenshot_20220228_123649](https://user-images.githubusercontent.com/72577339/156036080-b8678c06-9002-4747-8d74-344dfe5ca6b7.png)

On the other hand, while the image is been created there's some warning I do no know how to manage.
 
![Screenshot_20220228_122702](https://user-images.githubusercontent.com/72577339/156036553-b01f8477-220b-4964-bd92-64cbe41f1f00.png)

Reply:
Don't delete that line, it could create conflicts with definitive config files. Add the chown option to the copy statement as it is in the [frontend](https://github.com/PEM-Humboldt/biotablero-frontend/blob/develop/Dockerfile#L10) to solve that error. If it works, the best would be to add that same option to the other services' Dockerfiles

You can ignore the other warnings.

Reply:
Ok. All dockerfiles have now that option.